modules/functional-test/testng_integrationTests.xml

```diff
@@ -78,6 +78,11 @@
             <class name="com.intuit.wasabi.tests.service.segmentation.MutualExclusionBatchTest"/>
         </classes>
     </test>
+    <test name="IntegrationStatisticsTests">
+        <classes>
+            <class name="com.intuit.wasabi.tests.service.statistic.StatisticTest"/>
```

Comment:
@felixgao Awesome! I don't see the actual TestNG test class though. Not a part of this pull request? :)